bigbluebutton-html5/imports/ui/components/chat/message-form/component.jsx

```diff
@@ -123,8 +123,9 @@ class MessageForm extends Component {
           disabled={disabled}
           label={intl.formatMessage(messages.submitLabel)}
           hideLabel={true}
-          icon={"send"}
-          onClick={()=>{}}
+          icon={'send'}
+          onClick={()=> {}}
+
           />
```

Comment:
I wonder why linter didn't complain - this closing tag should be moved to the left. And a white space above removed..